README.Rmd

```diff
@@ -0,0 +1,191 @@
+is not exactly one, we use a different set of optimized scripts. This workflow
+is typically used to produce static (.pdf) reports. By contrast with the
+so-called "online" workflow, this one has been usually referred to as "the
+offline workflow"; again, this is misleading.
+
+Both workflows include three steps:
+
+1. Clean portfolio: Clean the input portfolio and merge in financial data, to
+categorize each holding and identify whether it's equity (EQ) or corporate bonds
+(CB).
+2. Run Analysis: Merge portfolios with asset-level data and scenarios, then
+group results at company, portfolio and regional level.
+3. Present results: Present the results in a clear output format.
+
+--
+
+Note: Code chunks with prompts "`$`" and "`#>`" correspond to bash and R,
+respectively.
+
+### Siblings
+
+PACTA_analysis depends on other "siblings" of the PACTA family. They are all
+available at <https://github.com/2DegreesInvesting> and should live alongside
+PACTA_analysis/, under the same parent directory. All siblings include an empty file called ".pacta", which you can use to find them programmatically.
+
+```{bash comment="$"}
+siblings="$(dirname $(find .. -maxdepth 2 -empty -type f -name '.pacta'))"
+echo $siblings
+```
+
+Before any analysis, ensure all siblings are up to date.
+
+```{bash comment="$"}
+# Store
+ working_directory="$pwd"
+
+siblings="$(dirname $(find .. -maxdepth 2 -empty -type f -name '.pacta'))"
+for i in $siblings; do
+  cd $i && echo $i
+  git log --decorate --oneline -n 1
+done
+
+# Restore
+cd "$working_directory"
+
+```
+
+For an easier way to work with all siblings at once from the terminal see
+[pacta-cli](https://github.com/2DegreesInvesting/pacta-cli).
+
+## Computing environment
+
+The required computing environment is available in [this docker
+image](https://github.com/2DegreesInvesting/docker/tree/master/r-packages).
+
+Other computing environments may be defined as workflows for GitHub actions.
+
+```{bash comment="$"}
+ls .github/workflows
+```
+
+Unless otherwise indicated, R packages should be up to date.
+
+<details> <summary>Session information.</summary>
+```{r}
+devtools::session_info()
+```
+</details>
+
+As an example, this is one way to use that computing environment to produce the
+very document you are now reading:
+
+* I move to the parent directory of PACTA_analysis and its siblings.
+
+```bash
+cd ~/git
+```
+
+* I run (`docker run`) an ephemeral (`--rm`), interactive (`-ti`) container of
+the image `2dii/r-packages`; create a number of volumes (`-v`) to access useful
+files in my host computer from inside the container; and set the working
+directory (`-w`) to PACTA_analysis/.
+
+```{r comment="", echo=FALSE}
+url <- "https://raw.githubusercontent.com/maurolepore/bin/master/docker-run-2dii-r-packages"
+tmp <- tempfile()
+download.file(url, tmp, quiet = TRUE)
+
+lines <- readLines(tmp)
+lines <- lines[2:length(lines)]
+writeLines(lines)
```

Comment:
that seems like a lot of hoop jumping just to display the command? I don't really understand what the purpose of that is